src/test/cls_sdk/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_executable(ceph_test_cls_sdk
+  test_cls_sdk.cc
+  )
+set_target_properties(ceph_test_cls_sdk PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
+target_link_libraries(ceph_test_cls_sdk
+  librados
+  global
+  ${EXTRALIBS}
+  ${BLKID_LIBRARIES}
+  ${CMAKE_DL_LIBS}
+  radostest
+  ${UNITTEST_LIBS}
+  )
+install(TARGETS
```

Comment:
@tchaikov I made the following changes and it works. Is there anything else that I would need to change?
1. Removed the install section and added
add_ceph_unittest(ceph_test_cls_sdk ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ceph_test_cls_sdk) in CMakeLists.txt.

2. Added qa/workunits/cls/test_cls_sdk.sh